pkgs/by-name/lu/luau-lsp/package.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, stdenv
+, lib
+, cmake
+, gcc9
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "luau-lsp";
+  version = "1.28.1";
+
+  src = fetchFromGitHub {
+    owner = "JohnnyMorganz";
+    repo = "luau-lsp";
+    rev = finalAttrs.version;
+    hash = "sha256-5+m5tLX0DRHrV6yI/SXc39orpAu1DlBmpNQ/rt7LUjQ=";
+    fetchSubmodules = true;
+  };
+
+  # Build fails if this is false
```

Comment:
do you know why ? seems like it could break several builds ?
Have you tried settings cmakeFlagsArray accordingly ?

Reply:
> do you know why ? seems like it could break several builds ?

If I set it to false the build fails with the error "Error: could not load cache"
log: https://gist.github.com/Eggflaw/795b8b25992eb976acf2de62ca4be702
I don't know why this happens but setting it true fixes the issue

> Have you tried settings cmakeFlagsArray accordingly ?

sorry but I am new to this. I don't know how to incoporate it

Reply:
should `dontUseCmakeConfigure` be false as much as possible? if I made it true,would there be any consequences from it?